README.md

```diff
@@ -1,20 +1,50 @@
-# UGM2023
-Materials for the [MDAnalysis User Group Meeting 2023](https://www.mdanalysis.org/pages/ugm2023/).
+# MDAnalysis UGM 2023
+The [MDAnalysis User Group Meeting](https://www.mdanalysis.org/pages/ugm2023/) (UGM) will bring together users of the MDAnalysis package from different communities. Our goal is to foster an opportunity to connect for interdisciplinary researchers and developers across biomolecular simulations, soft matter, biophysics and more. This repository includes relevant materials for the UGM.
```

Comment:
```suggestion
The [MDAnalysis User Group Meeting](https://www.mdanalysis.org/pages/ugm2023/) (UGM) will bring together users of the MDAnalysis package from different communities. Our goal is to foster an interdisciplinary opportunity to connect for researchers and developers across biomolecular simulations, soft matter, biophysics and more. This repository includes relevant materials for the UGM.
```

Just a minor suggestion, please ignore if you disagree!

Reply:
Incorporated this suggestion